flake8diff/main.py

```diff
@@ -129,7 +129,7 @@ def main():
             options=options,
         ).process()
     except Exception as e:
-        parser.error(six.text_type(e))
+        parser.error(six.text_type(str(e)))
```

Comment:
`six.text_type` is equivalent to `str` in Python3 and is equivalent to `unicode` in Python2 so no need to use `str` here